code/modules/mob/living/simple_animal/abnormality/waw/silence.dm

```diff
@@ -0,0 +1,60 @@
+//Coded by me, Kirie Saito!
+/mob/living/simple_animal/hostile/abnormality/silence
+	name = "The Price of Silence"
+	desc = "A scythe with a clock attached, quietly ticking."
+	icon = 'ModularTegustation/Teguicons/32x64.dmi'
+	icon_state = "silence"
+	threat_level = WAW_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(0, 0, 30, 40, 40),
+		ABNORMALITY_WORK_INSIGHT = 0,
+		ABNORMALITY_WORK_ATTACHMENT = list(0, 0, 10, 20, 30),
+		ABNORMALITY_WORK_REPRESSION = list(0, 0, 50, 45, 45)
+			)
```

Comment:
Gonna start trying to adjust our custom abnos work rates to be more in line with base game for at least "where they should be worked at" (level 4). This one has total work success of 105, the average WAW at that level has ~140. What I think is probably best is boosting these by 10 flat(excluding insight, clearly) and calling it a day. This is especially important because it does pale as well